lms/models/lti_params.py

```diff
@@ -173,6 +177,15 @@ def _apply_canvas_quirks(lti_params, request):
         "resource_link_id",
         [f"{CLAIM_PREFIX}/lti1p1", "resource_link_id"],
     ),
+    # LMS dependant variables that are not part of the "custom" claim
+    (
+        "org_defined_id",
```

Comment:
For context:

This maps one value in the LTI1.1 payload to the value in 1.3. But LTI1.3 values are often nested in a json structure so the right hand side of this is a list of keys.